generative_ai/context_caching/use_context_cache.py

```diff
@@ -33,6 +36,14 @@ def use_context_cache(project_id: str, cache_id: str) -> str:
     response = model.generate_content("What are the papers about?")
 
     print(response.text)
+    # Example response:
+    # The provided text is about a new family of multimodal models called Gemini, developed by Google.
+    # ...
+
     # [END generativeaionvertexai_gemini_use_context_cache]
 
     return response.text
+
+
+if __name__ == "__main__":
+    use_context_cache("6374036031984893952")
```

Comment:
minor-nit:  can you update this to like below to clearly tell that its an example number ? Please do the same for other samples too.


```suggestion
    use_context_cache("123456789")
```

Reply:
Done